ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -9,6 +9,7 @@ include 'quattor/schema';
 include 'pan/types';
 
 type uuid = string with match(SELF,'[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}');
```

Comment:
If it's used by more than on component then I'd say yes, do you feel like opening a new PR to add it to `pan/types` and another to remove it from the components?